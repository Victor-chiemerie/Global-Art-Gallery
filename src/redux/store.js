import { configureStore } from "@reduxjs/toolkit";
import reducer from './Slice'

export const store = configureStore({
    reducer: {
        art: reducer,
    },
});

export default store;