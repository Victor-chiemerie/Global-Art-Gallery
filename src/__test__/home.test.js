import React from 'react';
import { Provider } from 'react-redux'
import { BrowserRouter as Router } from "react-router-dom";
import renderer from 'react-test-renderer'
import Home from '../component/home';
import store from '../redux/store';

describe('Home', () => {
    it('renders correctly for reserved state', () => {
        const tree = renderer.create(
            <Provider store={store}>
            <Router>
            <Home />
            </Router>
            </Provider>
        );
        expect(tree).toMatchSnapshot();
    });
});