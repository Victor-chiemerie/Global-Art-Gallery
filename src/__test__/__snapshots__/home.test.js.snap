// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly for reserved state 1`] = `
<div>
  <p>
    idle
  </p>
  <div>
    Home page
  </div>
  <button
    onClick={[Function]}
    type="button"
  >
    Prev page
  </button>
  <button
    onClick={[Function]}
    type="button"
  >
    Next page
  </button>
</div>
`;
